src/v/cluster/cluster_utils.h

```diff
@@ -229,6 +229,10 @@ ss::future<std::error_code> replicate_and_wait(
        &as = as,
        timeout,
        use_serde_serialization](controller_stm& stm) mutable {
+          if (!stm.throttle(cmd)) {
```

Comment:
Do we throttle when replaying the log on start-up?

Reply:
No, we are replicating log. We don't produce requests during replaying, so throttling is not working there